src/components/Board/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import App from './App'
+import List from './list'
+import Card from './card'
+import Button from './Button'
+
+/**
+~~~~~~~~~~~ List Structure ~~~~~~~~~~~
+list: [{
+    title: "",
+    cards: [{
+        id: "",
+        text: "",
+        moveForward: () => {},
+        moveBackward: () => {},
+    }],
+}],
+}
+**/
+
+export default class Board extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {list: []}
+    }
+
+    render() {
+        return (
+            <h1> {this.props.name} </h1>
+            <div className='board-wrap'>
+                this.state.list.map((el) => (
+                    <List title={el.title} cards={el.cards} />
+                ))
+            </div>
+        )
+    }
+
+    move = (id, dir) => {
+        let index
+        let card
+        this.state.list.map((l, i) => l.filter(el => {
```

Comment:
filtering is nondestructive, so this won't actually modify the state. you should save this in a variable and use it to update the state